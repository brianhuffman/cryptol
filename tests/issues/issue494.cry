gt_words : {a} [a][32] -> [a][32] -> Bit
gt_words l0 l1 = ss!0
  where
    ss = [ False ] # [ s \/ (i > j) | i <- l0 | j <- l1 | s <- ss ]

gt_words' l0 l1 = ss!0
  where
    ss = [ False ] # [ s \/ (i > j) | i <- l0 | j <- l1 | s <- ss ]
