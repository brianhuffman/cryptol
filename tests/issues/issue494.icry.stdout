Loading module Cryptol
Loading module Cryptol
Loading module Main
gt_words' : {n, m, a} (Cmp a) => [n]a -> [m]a -> Bit
